examples/mhcaddi_examples.py

```diff
@@ -0,0 +1,26 @@
+"""Example with DeepSynergy."""
```

Comment:
fix docs... this doesn't seem very carefully done

Reply:
Fixed.